filebeat/module/system/auth/ingest/journald.yml

```diff
@@ -0,0 +1,204 @@
+description: Journald Pipeline for system/auth Filebeat module
+processors:
+  - set:
+      field: event.ingested
+      copy_from: _ingest.timestamp
+  - rename:
+      field: "journald.process.name"
+      target_field: process.name
+  - grok:
+      description: Grok specific auth messages.
+      tag: grok-specific-messages
+      field: message
+      ignore_missing: true
+      patterns:
+        - '^%{DATA:system.auth.ssh.event} %{DATA:system.auth.ssh.method} for (invalid user)?%{DATA:user.name} from %{IPORHOST:source.address} port %{NUMBER:source.port:long} ssh2(: %{GREEDYDATA:system.auth.ssh.signature})?'
+        - '^%{DATA:system.auth.ssh.event} user %{DATA:user.name} from %{IPORHOST:source.address}'
+        - '^Did not receive identification string from %{IPORHOST:system.auth.ssh.dropped_ip}'
+        - '^%{DATA:user.name} :( %{DATA:system.auth.sudo.error} ;)? TTY=%{DATA:system.auth.sudo.tty} ; PWD=%{DATA:system.auth.sudo.pwd} ; USER=%{DATA:system.auth.sudo.user} ; COMMAND=%{GREEDYDATA:system.auth.sudo.command}'
+        - '^new group: name=%{DATA:group.name}, GID=%{NUMBER:group.id}'
+        - '^new user: name=%{DATA:user.name}, UID=%{NUMBER:user.id}, GID=%{NUMBER:group.id}, home=%{DATA:system.auth.useradd.home}, shell=%{DATA:system.auth.useradd.shell}$'
+      ignore_failure: true
```

Comment:
I'm pretty sure this is a duplicate of what was in `pipeline.yml` which is now renamed to `files.yml`.   The grok patterns are hard enough to maintain, we don't want to have to remember to do it in 2 places.  Can you move the duplicates into a separate pipeline that both call?